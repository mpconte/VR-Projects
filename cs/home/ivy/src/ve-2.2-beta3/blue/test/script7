#
# Test constant parsing
#
number zero
number five
number nine
number one
number eight
number foobar
