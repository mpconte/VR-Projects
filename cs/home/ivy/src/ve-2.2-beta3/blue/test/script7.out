number = 0
number = 5
number = 9
number = 1
number = 8
result = error
error: unknown value for 'number': foobar
